plugins/utils_operators/agol_operators.py

```diff
@@ -186,6 +190,7 @@ def get_fields(self, query_url):
         params = "&".join([ f"{k}={v}" for k,v in query_string_params.items() ])
         url = "https://" + "/".join([ url_part for url_part in f"{query_url}/query?{params}".replace("https://", "").split("/") if url_part ])
         res = requests.get(url)
+        logging.info("Try and get fields: ", res)
```

Comment:
@garyqi-to this line causes an error that we've seen before in a previous PR you made.

You can't concatenate `res` to a string, since `res` is a requests "response" object

Reply:
Did you test these changes on a server yet?